meta-mbl-distro/recipes-core/systemd/systemd-net-conf.bb

```diff
@@ -10,11 +10,19 @@ LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecda
 
 SRC_URI = "file://mbl-hostname.sh \
            file://mbl-hostname.service \
-           file://10-mbl.network \
            file://ssh.dnssd \
            file://mdns.conf \
+           ${@bb.utils.contains('PACKAGECONFIG','development-dbg','file://10-mbl-development.network','',d)} \
+           ${@bb.utils.contains('PACKAGECONFIG','production-dbg','file://99-mbl-production.network','',d)} \
+           ${@bb.utils.contains('PACKAGECONFIG','production-dbg','file://10-mbl-production-dbg.network','',d)} \
+           ${@bb.utils.contains('PACKAGECONFIG','production-dbg','file://llmnr.conf','',d)} \
 "
 
+PACKAGECONFIG ??= "development-dbg"
+
+PACKAGECONFIG[development-dbg] = ""
+PACKAGECONFIG[production-dbg] = ""
```

Comment:
Discussed with @diego-sueiro in person. I don't think `development-dbg` and `production-dbg` are the best names for these config options. I think the names should describe what the option actually does.

Reply:
...Discussed with @diego-sueiro in person some more. Splitting up the different aspects of the network config into descriptive `PACKAGECONFIG` items would be a bit messy. And as Diego pointed out, this recipe is a config recipe itself so it's not too bad if it contains production v.s. development policy decisions.

Reply:
Changed in d8ccf66.